ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/toolbar/DiskResourceViewToolbarImpl.java

```diff
@@ -712,10 +715,10 @@ protected void onEnsureDebugId(String baseID) {
         moveMi.ensureDebugId(baseID + Ids.EDIT_MENU + Ids.MENU_ITEM_MOVE);
         editFileMi.ensureDebugId(baseID + Ids.EDIT_MENU + Ids.MENU_ITEM_EDIT_FILE);
         editInfoTypeMi.ensureDebugId(baseID + Ids.EDIT_MENU + Ids.MENU_ITEM_EDIT_INFO_TYPE);
-        metadataMi.ensureDebugId(baseID + Ids.EDIT_MENU + Ids.MENU_ITEM_METADATA);
-        copymetadataMi.ensureDebugId(baseID + Ids.EDIT_MENU + Ids.MENU_ITEM_METADATA
+        metadataMenu.ensureDebugId(baseID + Ids.EDIT_MENU + Ids.METADATA_MENU);
```

Comment:
Since you're moving the metadata menu items out from under the `Edit` Menu, I would update the metadata menu and menu items debug IDs so they don't have `Ids.EDIT_MENU` in them.